packages/react-aria-components/docs/advanced.mdx

```diff
@@ -115,6 +115,44 @@ function Stepper({children}) {
 
 The slots provided by each built-in React Aria component are shown in the Anatomy section of their documentation.
 
+#### Default slot
+
+The default slot is used to provide props to a component without specifying a slot name. This approach allows you to assign a default slot to a component for its default use case and enables you to specify a slot name for a specific use case.
+
+This example shows a custom `Button` component with slots for an icon with no slot name or an icon with a slot named "end".
```

Comment:
Small nitpick since the example doesn't have icons any longer
```suggestion
This example shows a custom component that passes a specific class name to a standard button child and to a button child with a slot named "end".

Reply:
Thanks